src/queries/test/graphql-requests.ts

```diff
@@ -382,5 +474,26 @@ export const POOL_METRICS_REQUEST = {
     poolTotalSwapVolume: true,
     poolTotalSwapFee: true,
     poolLiquidity: true,
+    tokenIn: true,
+    tokenAmountIn: true,
+    tokenOut: true,
+    tokenAmountOut: true,
+  },
+};
+
+export const TOKEN_PRICES_REQUEST = {
+  tokenPrices: {
+    __args: {
+      where: {
+        id_in: [
```

Comment:
If you think it helps:
```suggestion
        id_in: Object.values(TOKEN_SYMBOLS_TO_ADDRESS)
```

Reply:
I used the hardcoded values to make sure we're aware that those have changed (in case we changed them incorrectly) but happy to consider this, what do you think?

Reply:
I think it is a good ideea.